client/src/components/R&R/FactorSlider.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function FactorSlider({ name, value }) {
+  const percent = ((value - 1) / 4) * 100;
+
+  // positioning for triangle in slider
+  const minPosition = -0.5;
+  const maxPosition = 12.45;
+
+  return (
+    <div className="rr-factor-slider-container">
+      <h4>{name}</h4>
+      <div className="rr-factor-slider">
+        <span style={{ left: `${minPosition + (percent / 100) * (maxPosition - minPosition)}rem` }}>&#9660;</span>
+      </div>
+      <div>
+        <span className="rr-factor-slider-label">Too small</span>
+        <span className="rr-factor-slider-label" style={{ float: 'right' }}>Too large</span>
```

Comment:
These will need to eventually render depending on the specific factor. styling can be moved to style sheet? if not it isn't too distracting in this file.

Reply:
Ahh. Good catch - thx.  Will update for next PR.